test/uk/gov/hmrc/cdsreimbursementclaim/repositories/MongoTestSupport.scala

```diff
@@ -31,6 +31,7 @@ trait MongoTestSupport extends MongoSpecSupport with BeforeAndAfterEach with Bef
   abstract override def beforeEach(): Unit = {
     super.beforeEach()
     mongo().drop()
+    ()
```

Comment:
Why do you need this?

Reply:
This is your wart remover.
mongo.drop() doesn't return unit.
So I had to

Reply:
It doesn't complain for me.